apps/web/src/pages/auth/components/LoginForm.tsx

```diff
@@ -20,8 +19,16 @@ type LoginFormProps = {
   email?: string;
 };
 
+export interface LocationState {
+  redirectTo: {
+    pathname: string;
+  };
+}
+
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Why are we defining the type for the return of the `useLocation` hook here?‏

Reply:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** We should avoid exporting a generic type from here as we may unknowingly cause circular dependencies‏